.github/actions/spelling/allow.txt

```diff
@@ -0,0 +1,325 @@
+IntelliJ
+noopener
+acmedtssdstest
+admins
+adoptopenjdk
+afds
+wdm
+portile
+afdverify
+aks
+deno
+dra
+alex
+allejo
+IaaS
+Iaa
+pipeable
+SaaS
+PaaS
+haml
+WCAG
+sassc
+Tful
+RESTful
+hashie
+prebuilds
+acme
+acmebot
+racc
+TGL
+nta
+toc
+aad
+AAD
+aat
+rb-inotify
+acr
+ada
+adf
+alphagov
+parslet
+apastuhov
+ccd
+apim
+apps
+artifactory
+Frontdoors
+atlassian
+servolux
+Vnets
+autoconfigure
+@U1234ABCD
+ABCD
+nsenter
+autoscaler
+azuread
+tfvars
+azurecr
+azuredevops
+azureedge
+azurefd
+paa
+Intelli
+azurerm
+backports
+barrelsby
+basarat
+benlesh
+betterfm
+bfaac
+bitnami
+blobstorage
+blockquote
+brunton
+cdk
+cdn
+cdnverify
+certficates
+certs
+cfb
+cft
+cftapps
+cftprod
+cftptl
+checkstyle
+CJS
+cleardown
+cloudapp
+cnp
+codeburst
+codeowners
+codinghorror
+commonmarker
+contino
+continuousdelivery
+cppdev
+crictl
+customise
+cyclomatic
+datastore
+dba
+DBaa
+dburl
+dcd
+dce
+deserialise
+dts
+dtspo
+dynatrace
+envsubst
+erb
+erubis
+eventmachine
+excludesfile
+execjs
+expressjs
+fastimage
+ffi
+firebaseapp
+firefox
+firewalls
+fluxcd
+flywaydb
+fprofiles
+frontdoor
+frontends
+fsevent
+getbem
+gitbook
+gitops
+gitpod
+gitrepositories
+gkf
+googling
+gotchas
+gotk
+govuk
+gradle
+gradlew
+hackernoon
+helmrelease
+hmcts
+hmctsjobs
+hmctspublic
+hmi
+hotspot
+howto
+hpa
+idam
+imagepolicies
+imagepolicy
+imagerepositories
+img
+intsvc
+ithc
+itnext
+jattach
+javaguide
+jayne
+jcmd
+jdbc
+jdk
+jenkins
+jenkinsfile
+jfr
+jira
+jitpack
```

Comment:
Is this case sensitive, jira is `Jira` and jitpack is `JitPack` is may not complain if they are correctly capitalised.

Reply:
it is, I've resolved the `jira` instance, but jitpack is getting caught in a few places where it's referencing another existing file I think @timja

Reply:
is it possible to exclude in file names? seems a bunch of the words are in there because of that